docs/readme.md

```diff
@@ -8,22 +8,28 @@ Users authentication can be made via IMAP, LDAP or Microsoft Azure.
 - **app**: main AgentJ web interface (configuration for admins, usage for users)
 - **db**: a MariaDB instance to store mails, domains configuration, users info, DKIM keys, authorized/banned senders, Amavis scores …
 - **smtp**: a postfix instance that will receive the incoming e-mails and check them using **amavis** container
-- **relay**: a Postfix instance needed to avoid loops when forwarding the released or white-listed e-mails to their recipients(s)
+- **relay**: a postfix instance needed to avoid loops when forwarding the released or white-listed e-mails to their recipients(s)
 - **outsmtp**: a postfix instance that will handle outgoing e-mails, sent by local users (via their original smtp server) and check them using **outamavis**
 - **amavis**: a container running Amavis/Spamassassin and ClamAV services
 - **outamavis**: same as **amavis** but used for outgoing e-mails sent by local users
 - **opendkim**: verify incoming mail DKIM signature for incoming mail, and append signature for outgoing mail
 - **policyd-rate-limit**: rate limiting service used by **outsmtp**, get policies from **db**
-- *for tests only* **smtptest** and **badrelay**
+- **senderverifmilter**: custom postfix milter to secure multi-domains AgentJ instances
 
 > By default *ClamAV* run in the amavis containers, but you can run it externally (directly on the host or in a separated container). See `.env.example` for details
 
+### services for dev/tests
+
+- **mailpit** will catch all mail, from or to agentj
+- **smtptest** runs opensmtpd and dnsmasq. It can send mail to agentj on the behalf of configured domains; relay mail from agentj to mailpit, provide DNS for correct DKIM verification of tests mail
```

Comment:
> It can send mail to agentj on the behalf of configured domains

Je veux un peu de doc pour savoir comment envoyer des mails avec `swaks` (même si j’ai fini par trouver, ce sera utile à d’autres et à mon moi futur), et aussi comment bien configurer les domaines

Reply:
Un script qui soit utilisable et serve d'exemple ça le ferait ? Comme ça ça avance #132

Reply:
> Ça fait encore plus d'endroits à modifier si jamais il y avait un changement …

Ah non mais l’idée serait d’avoir le script et de documenter l’existence de ce script et de comment l’utiliser, pas de documenter en double.

> Je verrais plus une vraie doc d'exploitation […]

Mon idée était plutôt d'avoir un truc très simple du genre

> En phase de dev, pour ajouter un domaine, utilisez les paramètres "mailpit" et port "1025". Pour accédez aux emails envoyés par AgentJ, rendez-vous sur http://localhost:8025 (ou tout autre port que vous avez pu indiquer dans le `.env`)

Après, vraiment, là va au plus simple et rapide, j’aimerais bien merger ça aujourd'hui histoire de merger le reste ensuite plus tranquillement.

Reply:
Difficile de documenter ça simplement :smiling_face_with_tear: 

Dis moi si c'est incompréhensible et si tu vois quoi améliorer, mais sinon on merge et on affinera la doc ensuite. De toute façon j'ouvre une issue pour documenter senderverifmilter, c'était passé à la trappe